packages/dcos-integration-test/extra/test_mesos.py

```diff
@@ -53,8 +38,10 @@ def test_if_marathon_app_can_be_debugged(dcos_api_session):
     app_id = 'integration-test-{}'.format(test_uuid)
     with dcos_api_session.marathon.deploy_and_cleanup(app):
         # Fetch the mesos master state once the task is running
-        master_state_url = 'http://{}:{}/state'.format(dcos_api_session.masters[0], 5050)
-        r = requests.get(master_state_url)
+        scheme = dcos_api_session.default_url.scheme
+        master_ip = dcos_api_session.masters[0]
+        master_state_url = '{}://{}:{}/state'.format(scheme, master_ip, 5050)
```

Comment:
If you look at where I use `master_state_url`, you'll notice this is purely for logging. The original code logged the full URL, I decided to keep that behaviour to make this PR as non-invasive as possible.